pkg/odo/cli/registry/update.go

```diff
@@ -0,0 +1,97 @@
+package registry
+
+import (
+	// Build-in packages
+	"fmt"
+
+	// Third-party packages
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
+
+	// odo packages
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/odo/cli/ui"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/openshift/odo/pkg/preference"
+)
+
+const updateCommandName = "update"
+
+// "odo registry update" command description and examples
+var (
+	updateLongDesc = ktemplates.LongDesc(`Update devfile registry`)
```

Comment:
We should clarify here that it helps to update only the URL of the devfile registry. It can't help change the name of the registry. Or does it?

Reply:
I can update the message to make it more clear.